.gitignore

```diff
@@ -61,4 +61,7 @@ target/
 *~
 
 # Pycharm stuff
-.idea/
\ No newline at end of file
+.idea/
```

Comment:
This should really go in your ~/.gitignore file BTW, but I know it was here already.